app/models/review.rb

```diff
@@ -22,7 +22,7 @@ class Review < ActiveRecord::Base
 
 
   def self.default_load
-    self.includes({ :associate_consultant => :user })
+    self.includes({ :associate_consultant => :user }, { :associate_consultant => :reviewing_group })
```

Comment:
Line is too long. [100/80]<br>Use the new Ruby 1.9 hash syntax.<br>Redundant `self` detected.